### ISSUE TITLE
More than one "H1" heading levels are provided to the ONNX Runtime landing page: A11y_WCP URLs - ONNX Runtime_ONNX Runtime_Home_AI4W.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20096]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCAG1.3.1; #WCP; #Win11;#FTP; #P3_WCP; #Element:Heading; #ONNX Runtime; #Info and Relationships; #ChromiumEdge; #A11yMAS;

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".** 

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2

### Steps to Reproduce

1. Hit the URL "https://onnxruntime.ai/".
2. "ONNX Runtime" home page is appeared.
3. Turn on the headings toggle in accessibility Insights for web.
4. Now check that more than one "H1" heading levels are provided to the ONNX Runtime landing page.

### Actual Results:
More than one "H1" heading levels are provided to the ONNX Runtime landing page.
### Expected Results:
Headings should be provided on the page to help users navigate the page and understand the structure of the content. Headings should be properly formatted and follow a logical hierarchy. But, in a page only one "H1" level heading should be provided.
### User Impact:
People with cognitive disabilities may have difficulty understanding the content of the page without proper hierarchy structure of headings. Additionally, people who rely on assistive technologies such as screen readers may not be able to navigate the page without proper hierarchy structure of headings.

### Attachment:

[H1 heading levels.zip](https://github.com/microsoft/onnxruntime/files/14788705/H1.heading.levels.zip)

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.97 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is not fixed we have two h1 level is there in a page. Hence the bug is Reopened.
#Regressed:05-07-24; #Reopened;

For reference attachment is added.

[Recording-20240507_152024.webm](https://github.com/microsoft/onnxruntime/assets/142479541/37f5e806-759c-4bd1-bf6b-06b074258e47)

**Comment by PadminiGariki** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/training
Edge version:125.0.2535.92 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
For reference attachment is added.

#Regressed:06-17-24; #Closed;

### Attachment:

[Recording-20240617_102921.webm](https://github.com/microsoft/onnxruntime/assets/142479541/dc42a8ca-4a58-4fdd-b780-771daa558649)
